env/dev/web/index.php

```diff
@@ -1,9 +1,12 @@
 <?php
 defined('YII_DEBUG') or define('YII_DEBUG', true);
 defined('YII_ENV') or define('YII_ENV', 'dev');
+defined('COMPOSER_VENDOR_DIR') or define ('COMPOSER_VENDOR_DIR', __DIR__ . '/../vendor');
```

Comment:
What the purpose of defining it?

```suggestion
defined('COMPOSER_VENDOR_DIR') or define('COMPOSER_VENDOR_DIR', __DIR__ . '/../vendor');
```

Reply:
Otherwise `vendor` folder will be missing after `docker_compose up` (in case of being inside shared volume which is in current folder). Or you need to keep a copy of it on host machine which ruins the whole concept. The same with `node_modules`. 

However I think I'll choose a slightly different approach for achieving this because currently there are some collisions with other compose install (for example during API docs generation). It's still work in progress.

Reply:
Example for `node_modules`:

- https://stackoverflow.com/questions/30043872/docker-compose-node-modules-not-present-in-a-volume-after-npm-install-succeeds
- https://stackoverflow.com/a/33106611/4323648